libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/DefaultAudioTrackBufferSizeProvider.java

```diff
@@ -62,8 +62,16 @@ public class DefaultAudioTrackBufferSizeProvider
    * devices (e.g., Broadcom 7271).
    */
   private static final int AC3_BUFFER_MULTIPLICATION_FACTOR = 2;
+  /**
+   * Default multiplication factor to apply to DTS Express passthrough buffer to avoid underruns
+   * on some devices (e.g., Xiaomi A2 TV).
+   */
+  private static final int DTSHD_BUFFER_MULTIPLICATION_FACTOR = 4;
+
+  /**
+   * A builder to create {@link DefaultAudioTrackBufferSizeProvider} instances.
```

Comment:
nit: Since there is no actual required diff at lines 71-74, could you please revert these lines back to the original state?

Reply:
Done